libs/framework/gtest/src/DependencyManagerTestSuite.cc

```diff
@@ -114,6 +114,10 @@ TEST_F(DependencyManagerTestSuite, StartDmWillBuildCmp) {
 }
 
 struct TestService {
+public:
+    TestService() : handle(nullptr) {};
+    TestService(void *_handle) : handle(_handle) {};
```

Comment:
It's a bit of a nitpick, but these constructors are not necessary. It's a [best practice](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rc-zero) to not define constructors/destructors if possible. Modify line 121 to `void *handle{nullptr};` instead.

Consequently, modify all initializations of TestService to `TestService svc{};` (e.g. line 162).

Reply:
Thanks, I've edited the implementation.